src/samples/Authorization/AutoSignIn/appsettings.json

```diff
@@ -0,0 +1,54 @@
+{
+  "AgentApplication": {
+    "StartTypingTimer": true,
+    "RemoveRecipientMention": false,
+    "NormalizeMentions": false,
+
+    "UserAuthorization": {
+      "Default": "graph",
+      "AutoSignin": true,
```

Comment:
are config keys case insensitive? I believe it is `AutoSignIn` from https://github.com/microsoft/Agents-for-net/blob/55b4ff799be58f5504987a6f01d5dbfe4fa03519/src/libraries/Builder/Microsoft.Agents.Builder/App/UserAuth/UserAuthorizationOptions.cs#L42

Reply:
Appsettings are not case sensitive.